eclass/node-guru.eclass

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EXPORT_FUNCTIONS src_prepare src_compile src_install
+
+SLOT="0"
+IUSE=""
+DEPEND="net-libs/nodejs"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	app-misc/jq
+	net-misc/rsync
+	sys-apps/moreutils
+"
```

Comment:
```suggestion
NODEJS_DEPEND="net-libs/nodejs"
NODEJS_BDEPEND="
	app-misc/jq
	net-misc/rsync
	sys-apps/moreutils
"
DEPEND="${NODEJS_DEPEND}"
RDEPEND="${NODEJS_DEPEND}"
BDEPEND="${NODEJS_BDEPEND}"
```
maybe something like this so it can be overridden and called from in ebuilds?

Reply:
See the new changes